src/public/login/index.html

```diff
@@ -6,16 +6,23 @@
     <link rel="stylesheet" href="login.css">
   </head>
   <body>
+   
     <div class="container">
       <h1>Harbor</h1>
       <form id="form-login">
-        <label for="username">Username</label>
-        <input type="number" id="username">
-        <label for="password">Password</label>
-        <input type="password" id="password">
-        <input type="submit" id="btn-login" value="Login">
+        <div class="input-group">
```

Comment:
aren't all these Bootstrap 3 classes? We're not using Bootstrap 3, nor do we plan to. Did you even check this out before trying? Is this a spam PR?